node/src/components/consensus/highway_core/highway/vertex.rs

```diff
@@ -59,32 +62,29 @@ impl<C: Context> Vertex<C> {
     pub(crate) fn value(&self) -> Option<&C::ConsensusValue> {
         match self {
             Vertex::Unit(swunit) => swunit.wire_unit().value.as_ref(),
-            Vertex::Evidence(_) | Vertex::Endorsements(_) => None,
+            Vertex::Evidence(_) | Vertex::Endorsements(_) | Vertex::Ping(_) => None,
         }
     }
 
     /// Returns the unit hash of this vertex (if it is a unit).
     pub(crate) fn unit_hash(&self) -> Option<C::Hash> {
         match self {
             Vertex::Unit(swunit) => Some(swunit.hash()),
-            Vertex::Evidence(_) | Vertex::Endorsements(_) => None,
+            Vertex::Evidence(_) | Vertex::Endorsements(_) | Vertex::Ping(_) => None,
         }
     }
 
     /// Returns whether this is evidence, as opposed to other types of vertices.
     pub(crate) fn is_evidence(&self) -> bool {
-        match self {
-            Vertex::Unit(_) | Vertex::Endorsements(_) => false,
-            Vertex::Evidence(_) => true,
-        }
+        matches!(self, Vertex::Evidence(_))
     }
 
     /// Returns a `Timestamp` provided the vertex is a `Vertex::Unit`
     pub(crate) fn timestamp(&self) -> Option<Timestamp> {
         match self {
             Vertex::Unit(signed_wire_unit) => Some(signed_wire_unit.wire_unit().timestamp),
-            Vertex::Evidence(_) => None,
-            Vertex::Endorsements(_) => None,
+            Vertex::Ping(ping) => Some(ping.timestamp()),
```

Comment:
The comment could be updated. Now it only mentions `Vertex::Unit`.

Reply:
I don't think the comment has been updated.

Reply:
Yes it has. GitHub just shows the snippet of the version you commented on.